hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/IncrementalContainerReportHandler.java

```diff
@@ -87,7 +89,7 @@ public void onMessage(final IncrementalContainerReportFromDatanode report,
             report.getDatanodeDetails(), ex);
       } catch (IOException e) {
         success = false;
-        LOG.error("Exception while processing ICR for container {}",
+        LOG.warn("Exception while processing ICR for container {}",
```

Comment:
I think rather than changing here, we should catch a separate `ContainerReplicaNotFoundException ` and there reduce the log level, suppressing for all IOE might not be a good choice.
@jojochuang does that make sense?

Reply:
@wycccccc can you address this as well

Reply:
@ayushtkn I think what you said makes sense，let me try it.